packages/paste-website/src/component-examples/AIChatLogExamples.ts

```diff
@@ -0,0 +1,302 @@
+export const basicBotMessage = `
+const BasicMessage = () => {
+  return (
+    <AIChatLog>
+        <AIChatMessage variant="bot">
+          <AIChatMessageAuthor aria-label="AI said">Good Bot</AIChatMessageAuthor>
+          <AIChatMessageBody>
+            Here is what I found, error code 30003 means: The destination phone is unavailable or turned off, or it may be a landline or phone that doesn't support SMS.
+          </AIChatMessageBody>
+        </AIChatMessage>
+    </AIChatLog>
+  );
+};
+
+render(
+  <BasicMessage />
+)`.trim();
+export const basicHumanMessage = `
+const BasicMessage = () => {
+  return (
+    <AIChatLog>
+        <AIChatMessage variant="user">
+          <AIChatMessageAuthor aria-label="You said at 2:36pm">Gibby Radki</AIChatMessageAuthor>
+          <AIChatMessageBody>
+            I would like some information on twilio error codes for undelivered messages
+          </AIChatMessageBody>
+        </AIChatMessage>
+    </AIChatLog>
+  );
+};
+
+render(
+  <BasicMessage />
+)`.trim();
+export const botWithFeedback = `
+const MessageWithFeedback = () => {
+  return (
+    <AIChatLog>
+      <AIChatMessage variant="bot">
+        <AIChatMessageAuthor aria-label="AI said">Good Bot</AIChatMessageAuthor>
+        <AIChatMessageBody>
+          Here is what I found, error code 30003 means: The destination phone is unavailable or turned off, or it may be a landline or phone that doesn't support SMS.
+        </AIChatMessageBody>
+        <AIChatMessageActionGroup aria-label="Feedback form">
+          <AIChatMessageActionCard>
+            Is this helpful?
+            <Button variant="reset" size="reset">
+              <ThumbsUpIcon decorative={false} title="like result" />
+            </Button>
+            <Button variant="reset" size="reset">
+              <ThumbsDownIcon decorative={false} title="dislike result" />
+            </Button>
+          </AIChatMessageActionCard>
+          <AIChatMessageActionCard>
```

Comment:
suggestion: I think I added some aria labels to the rewrite button and maybe also the thumbs up/down buttons in the stories - can you copy whatever I did there for the action card sections in these examples?

Reply:
Great catch. I will address this in the next commit